extensions/sql-migration/src/constants/strings.ts

```diff
@@ -332,6 +332,7 @@ export function MI_CONFIGURATION_PREVIEW(hardwareType: string, computeTier: stri
 }
 export const GENERAL_PURPOSE = localize('sql.migration.sku.azureConfiguration.generalPurpose', "General purpose");
 export const BUSINESS_CRITICAL = localize('sql.migration.sku.azureConfiguration.businessCritical', "Business critical");
+export const NEXTGEN_GENERAL_PURPOSE = localize('sql.migration.sku.azureConfiguration.nextGenGeneralPurpose', "Next-Gen General Purpose");
```

Comment:
It should be "Next-gen  General Purpose", please check with Ajith once.

Reply:
<img width="385" alt="image" src="https://github.com/microsoft/azuredatastudio/assets/89977091/93bc5afe-0140-402c-9092-1e76bf45c516">

Reply:
done